ci/github-script/reviewers.js

```diff
@@ -0,0 +1,103 @@
+// Caching the list of team members saves API requests when running the bot on the schedule and
+// processing many PRs at once.
+const members = {}
+
+async function handleReviewers({
+  github,
+  context,
+  core,
+  log,
+  dry,
+  pull_request,
+  reviews,
+  maintainers,
+  owners,
+  getTeamMembers,
+  getUser,
+}) {
+  const pull_number = pull_request.number
+
+  const users = new Set([
+    ...(await Promise.all(
+      maintainers.map(async (id) => (await getUser(id)).login),
+    )),
+    ...owners.filter((handle) => !handle.includes('/')),
+  ])
+
+  const teams = new Set(
+    owners
+      .map((handle) => handle.split('/'))
+      .filter(([org, slug]) => org === context.repo.owner && slug)
+      .map(([, slug]) => slug),
+  )
+
+  const team_members = new Set(
+    (await Promise.all(Array.from(teams).map(getTeamMembers)))
```

Comment:
We can simplify `Array.from(it).map(fn)` to `Array.from(it, fn)` in a few places